File: src/main/java/me/timtim17/dev/pscsta/spring2015/java/SayHello.java
package me.timtim17.dev.pscsta.spring2015.java;

public class SayHello {
    public static void main (String[] args) {
        String[] names = {"Austin", "Myself", "I"};

        int letters = 0;
        for (String name : names) {
            letters += name.length();
        }

        System.out.print("Hello judges! Our names are ");
        for (int i = 0; i < names.length; i++) {
            if (i == names.length - 1) {
                System.out.print("and " + names[i] + ". ");
            } else {
                System.out.print(names[i] + ", ");
            }
        }

        System.out.println("There are " + letters + " letters in our names. Have a wolf strong day! Wolf Strong Pack Strong!");
    }
}

File: SayHello.py
names = ["Austin", "Daniel"]
length = reduce(lambda x, y: len(x) + len(y), names)

print "Hello Great and Grand Judges! We are %s and %s. We go to Eastlake, where every day we have %d letters in our names combined. Have a wolf strong day!" % (names[0], names[1], length)
